src/vm/exceptionhandling.cpp

```diff
@@ -5156,7 +5156,7 @@ static BOOL IsIPinVirtualStub(PCODE f_IP)
     }
 
     VirtualCallStubManager::StubKind sk;
-    VirtualCallStubManager::FindStubManager(f_IP, &sk);
+    VirtualCallStubManager::FindStubManager(f_IP, &sk, false /* usePredictStubKind */);
```

Comment:
@janvorli Should I use ``FALSE`` (instead of ``false``) here?

Reply:
Yes